docs/src/main/paradox/whygrpc.md

```diff
@@ -0,0 +1,47 @@
+# Why gRPC?
```

Comment:
Added page "Why gRPC?" that contains the previous comparison + some extra motivation for gRPC.